search-service/src/main/kotlin/com/egm/stellio/search/web/TemporalEntityHandler.kt

```diff
@@ -120,7 +120,12 @@ class TemporalEntityHandler(
             jsonLdEntityWithTemporalValues.contexts
         )
 
-        return ResponseEntity.status(HttpStatus.OK).body(serializeObject(filteredJsonLdEntity.compact()))
+        // WORKAROUND:
+        // the json-ld lib shows empty attributes as empty objects but we need empty arrays as stated in 5.7.4.4
+        // replace empty Maps by empty Lists
+        val postProcessedEntity = filteredJsonLdEntity.compact().replaceEmptyMapsByLists()
```

Comment:
I don't really understand why there is a first process in TemporalEntityAttributeService and another one here.